### DISCUSSION TITLE
How to configure Telegram Notifications

### DISCUSSION THREAD

**Discussion by M1XZG** [upptime/upptime#763] [General] [answered]
I've been through the docs and have created the 3 required variables. for the `chat_id` I've tried both my personal ID (not username) as well as a group chat ID which works with other scripts I have running. 

Am I right in thinking that simply setting `NOTIFICATION_TELEGRAM`=`true` is all that's required to enable this notification?  It feels like I'm missing something. 

![Snag_682747c](https://user-images.githubusercontent.com/22931360/232258498-0258f2fb-659e-4c4a-a74d-a9da056fb7d4.png)

Thanks in advance for any guidance on this.

**Comment by M1XZG** [comment, marked as answer]
I actually worked it out.. the wording on the [notifications](https://upptime.js.org/docs/notifications/) page is kind of awkward so it wasn't clear.

I'd used `Repo Variable` but it should have been `Repo Secrets`. Once I moved all the variables over everything worked. 

I hope this info helps someone. I've opened a [pull request](https://github.com/upptime/upptime.js.org/pull/41) to fix the docs and make it a bit more clear for the future.

**Reply by EisenwerkHasenclever** [reply]
Hello M1XZG.

I also tried to configure my telegram notifications.

![image](https://github.com/upptime/upptime/assets/137895844/e81b4906-c3ef-47cf-b0d4-d4f409e51557)

I also configured the variables in the right place.

Do i have to make any further configuration or is it done with just setting up the variables?

I dont get any messages on telegram

**Comment by M1XZG** [comment]
> I also configured the variables in the right place.
> 
> Do i have to make any further configuration or is it done with just setting up the variables?
> 
> I dont get any messages on telegram

That should be all that's required. As long as the contents of the 3 secrets are correct.  I don't have it handy but there are some simple scripts that can use those variables to test that your bot is listening and accepting posts. 

Also, have a look through the logs of a run that should have sent a message to telegram, there maybe something it the workflow log to give a clue.  If you have the workflow run URL where it didn't send to telegram I can have a look.
